packages/http/src/validator/validators/__tests__/headers.spec.ts

```diff
@@ -66,6 +66,19 @@ describe('HttpHeadersValidator', () => {
 
                 expect(validateAgainstSchemaModule.validateAgainstSchema).toHaveBeenCalled();
               });
+
+              it('ignores the header casing', () => {
```

Comment:
we should probably add a similar test that has `required: false`

Reply:
If required is false the check is not happening at all. What would you like me to test in particular?

Reply:
Hm.. ok, so maybe I don't understand. There now is:
`if (!(spec.name in loweredCaseTarget) && spec.required === true)`. And later there is `if (spec.content && spec.content.schema && target[spec.name])`. Don't we need a test for `if (spec.content && spec.content.schema && target[spec.name])`?